docs/rest/rest-catalog-open-api.yaml

```diff
@@ -661,12 +647,22 @@ components:
       properties:
         overrides:
           type: object
+          uniqueItems: true
```

Comment:
As this is a `Map` (or the type is `object`), I'm not sure if `uniqueItems` provides much value.

Open to removing it if anybody thinks I should.

Reply:
Yeah, probably not useful. It just makes me wonder if it isn't required for all the other maps.

Reply:
Removed.